src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar.domain
+
+class Cars(val cars: List<Car>) {
+    val size: Int
+        get() {
+            return cars.size
+        }
+
+    fun driveCars() {
+        cars.forEach { it.drive() }
+    }
+
+    fun getWinners(): String {
+        val maxPosition = cars.maxOf { it.distance }
+        val winners = cars.filter { it.distance == maxPosition }
+        return winners.joinToString(",") { it.name.name }
+    }
```

Comment:
> 질문사항
> 일급 컬렉션 사용 중 값을 호출할 때 중복된 이름을 계속 사용하게 되었습니다..
> 
>    fun getWinners(): String {
>         val maxPosition = cars.maxOf { it.distance }
>         val winners = cars.filter { it.distance == maxPosition }
>         return winners.joinToString(",") { it.name.name }  // 이렇게 name.name 으로 이름이 겹칩니다..
>     }
> 이럴때 어떻게 해결할지 좋은 생각이 떠오르지 않아 질문드립니다!!

it.name.name 이라는 중복 된 부분이 원시 값을 포장하면 생길 수 있는 부작용이긴합니다.  저는 이때 Car class에 getName()라는 함수를 만들어서 호출해오는 방식을 사용하고 있습니다. 이렇게 되면 it.name.name이 it.getName()이 되겠죠? 

단, 현재 로직은 view와 domain 로직이 섞여 보이는데요. 차라리 Cars라는 응답을 보내고 view에서 꺼내쓰는 건 어떨까요?
view에서 꺼내쓰는 것은 train wreck한 코드도 써도 된다고 생각합니다.

```suggestion
    fun getWinners(): Cars {
        val maxPosition = cars.maxOf { it.distance }
        val winners = cars.filter { it.distance == maxPosition }
        return Cars(winners)
    }
```